bin/test-cleanup

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/local/bin/bash
```

Comment:
Why `/usr/local`?

Reply:
Whoops I didn't mean to include this in the changes!